Documentation/ExtensionArchitecture/Concepts/SystemAndLocalExtensions.rst

```diff
@@ -1,55 +1,59 @@
 .. include:: /Includes.rst.txt
 
-
 .. _extension-scope:
 
 ===============================
 System and Local Extensions
 ===============================
 
-The files for an extension are located in a folder named by the
-*extension key*. The location of this folder can be either inside
-`typo3/sysext/` or `typo3conf/ext/`.
-
-The extension *must* be programmed so that it does automatically
-detect where it is located and can work from both locations.
-
+The files for an extension are located in a folder named vendor, followed by
+a folder with the composer name. The location of this folder is in the directory
+:ref:`vendor/ <directory-vendor>`.
 
+In legacy installations they are found in `typo3/sysext/` or `typo3conf/ext/`.
 
 .. _extension-local:
 
 Local Extensions
 ================
 
-Local extensions are located in the :file:`typo3conf/ext/` directory.
+Local extensions have the composer type `typo3-cms-extension`:
 
-This is where to put extensions *which are local* for a particular
-TYPO3 installation. The :file:`typo3conf` directory is always local, containing
-local configuration (e.g. :file:`LocalConfiguration.php`), local modules etc.
-If you put an extension here it will be available for a single TYPO3
-installation only. This is a "per-database" way to install an
-extension.
+..  code-block:: json
+    :caption: EXT:my_extension/composer.json`
 
+    {
+        "name": "myvendor/my-extension",
+        "type": "typo3-cms-extension",
+        "...": "..."
+    }
+
+The extension will be installed in the directory :ref:`vendor <directory-vendor>`
```

Comment:
The sentence sounds bumpy. Also: We should differentiate between stored (by the developer) and installed (by Composer). It is not obvious what is meant here.

Also: "vendor" is not linked.

Reply:
Rephrased it